pkg/containermeta/collectors/kubelet/kubelet.go

```diff
@@ -0,0 +1,97 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package kubelet
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/containermeta"
+	"github.com/DataDog/datadog-agent/pkg/util/kubernetes/kubelet"
+)
+
+const (
+	collectorID = "kubelet"
+	expireFreq  = 5 * time.Minute
+)
+
+type collector struct {
+	watcher    *kubelet.PodWatcher
+	ch         chan []containermeta.Event
+	lastExpire time.Time
+	expireFreq time.Duration
+}
+
+func init() {
+	containermeta.RegisterCollector(collectorID, func() containermeta.Collector {
+		return &collector{}
+	})
+}
+
+func (c *collector) Start(ctx context.Context, ch chan []containermeta.Event) error {
+	if !config.IsKubernetes() {
```

Comment:
nit: it should be using Env AD results.

Reply:
Copied it from the existing kubelet collector for the tagger, it was using this. Updated both here and the tagger collector now 👍